### PR TITLE
doc: prepare new release 3.4.2

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
related to #4011 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] No manual test necessary

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## 3.4.2 / 2021-01-07
+
+### Fixed Issues
+
+- UX: [4023](https://github.com/tracim/tracim/issues/4023)
+- Backend: [3997](https://github.com/tracim/tracim/issues/3997),
+[4000](https://github.com/tracim/tracim/issues/4000)
+- Doc: [3988](https://github.com/tracim/tracim/issues/3988)
```
Comment (grignards) [inline changelog.md:8]:
```suggestion
- UX: [#4023](https://github.com/tracim/tracim/issues/4023)
- Backend: [#3997](https://github.com/tracim/tracim/issues/3997),
[#4000](https://github.com/tracim/tracim/issues/4000)
- Doc: [#3988](https://github.com/tracim/tracim/issues/3988)
```

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+## 3.4.2 / 2021-01-07
+
+### Fixed Issues
+
+- UX: [4023](https://github.com/tracim/tracim/issues/4023)
```
Comment (grignards) [inline changelog.md:5]: Is this an issue? Or rather a new functionality?

Comment (PhilippeAccorsi) [reply changelog.md:5]: good question

Comment (raphj) [reply changelog.md]: it's a feature

File: changelog.md (hunk 1 above)
Comment (PhilippeAccorsi) [reply changelog.md]: Fix also for 3.4.1

File: changelog.md (hunk 2 above)
Comment (PhilippeAccorsi) [reply changelog.md]: thanks, I have not really read isse label :(

File: changelog.md [hunk 3]
```diff
@@ -1,8 +1,21 @@
+## 3.4.2 / 2021-01-08
+
+### New Features
+
+- Personal activity feeds is now the home page when you login in Tracim if you have space
```
Comment (grignards) [inline changelog.md:5]:
```suggestion
- The personal activity feed is now the home page when you log in Tracim when you are a member of a space
```

**Comment by PhilippeAccorsi** [comment]
@grignards I have validate your proposal

**Review by raphj — APPROVED** [review]

**Pull request merged**